examples/StandaloneMain/main_opengl_single_example.cpp

```diff
@@ -93,9 +95,8 @@ int main(int argc, char* argv[])
 		app->m_instancingRenderer->init();
 		app->m_instancingRenderer->updateCamera(app->getUpAxis());
 
-		btScalar dtSec = btScalar(clock.getTimeInSeconds());
-		if (dtSec < 0.1)
-			dtSec = 0.1;
```

Comment:
The real bug was at old line 97: the `<` comparison operator should go the other direction.  Should a github issue be added for bug tracking?  Or is that unnecessary?

Reply:
Thanks for the contribution. Can you just swap the > sign and undo the other changes? I'd rather keep the change minimal, and not start clamping by default in the CommonRigidBodyBase.h